### DISCUSSION TITLE
모집중 & 모집완료로 필터링해서 정보 조회?

### DISCUSSION THREAD

**Discussion by tco0427** [woowacourse-teams/2022-moamoa#48] [토론]
Q. 모집중, 모집완료 (OPEN, CLOSE) 로 필터링해서 정보를 조회할 수 있게 할 것인가??

(현재 백엔드는 모집중, 모집완료를 Study의 status로써 관리하고 있는데, 이 또한 하나의 필터정보(4기, Spring)으로 볼 것인가?
정렬시에 최신순으로 정렬하면 어차피 모집중이 먼저 나오고 나서 모집완료된 항목들이 나오지 않을까?)

**Comment by jaejae-yoo** [comment]
모집중이 모집완료보다 뒤에 있을 경우(모집 기간이 상대적으로 긴 스터디)도 있을 것 같아서 필터링하면 좋을 것 같아요!

**Comment by nan-noo** [comment]
필터 항목에 추가하는 것도 좋습니다!
올리브영같은 곳은 아예 검색 결과 리스트를 보여줄 때 품절 상품이나 재고 없는 상품은 후순위로 미뤄놓던데, 필터를 안 쓰면 이런 방법도 좋을 것 같아요!

**Reply by jaejae-yoo** [reply]
오 필터링 안해도 모집완료 된 것은 후순위로 보이게 하는 방법 좋은 것 같아요 👍

**Comment by verus-j** [comment]
일단 위 기능은 추후 생각하는게 좋을 것 같아요. 현재 필터링 기능만으로 스터디 조회 관련에 대해 모아모아의 가치를 높이는 데 충분하다고 생각합니다.
검색 시 기본으로 최신순 정렬을 하면 더욱 위의 기능은 가치가 많이 떨어지겠죠... 그것말고 이제 CUD를 고려해봅시다!
